diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4493131..d8778d2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -208,9 +208,9 @@ endif()
 if("${TARGET}" STREQUAL "X86" OR "${TARGET}" STREQUAL "X86_DARWIN")
     add_assembly(${FFI_ROOT}/src/x86/sysv.S)
 elseif("${TARGET}" STREQUAL "X86_64")
-#    if(NOT MSVC AND NOT MINGW)
+    if(NOT MSVC AND NOT MINGW)
         add_assembly(${FFI_ROOT}/src/x86/unix64.S)
-#    endif()
+    endif()
     add_assembly(${FFI_ROOT}/src/x86/win64.S)
 elseif("${TARGET}" STREQUAL "X86_WIN32")
     if(MSVC)
